lib/virt_autotest/utils.pm

```diff
@@ -116,6 +117,42 @@ sub is_kvm_host {
     return check_var("SYSTEM_ROLE", "kvm") || check_var("HOST_HYPERVISOR", "kvm") || check_var("REGRESSION", "qemu-hypervisor");
 }
 
+#retrun 1 if libvirt 9.0- is running which monolithic libvirtd is the default service
+sub is_monolithic_libvirtd {
+    unless (is_alp) {
+        return 1 if script_run('rpm -q libvirt-libs | grep -e "libs-9\.0" -e "libs-[1-8]\."') == 0;
+    }
+}
+
+# For legacy libvird, set debug level logging for libvirtd services
+# For modular libvirt, do the same settings to /etc/libvirt/virt{qemu,xen,driver}d.conf.
+# virt{qemu,xen}d daemons provide the most important libvirt log(sufficient for most issues).
+# virt{driver}.d daemons is only required by specific issues, eg virtual network failures may need virtnetworkd log.
+# But our automation is better to set them to collect more logs as we could as possible.
+# Developer asked to use different log file as log_output per daemon.
+sub turn_on_libvirt_debugging_log {
+
+    my @libvirt_daemons = is_monolithic_libvirtd ? "libvirtd" : qw(virtqemud virtstoraged virtnetworkd virtnodedevd virtsecretd virtproxyd virtlockd virtlogd);
```

Comment:
For virtlockd and virtlogd, suggest to let other virt{drv}d start/stop/restart them by dependency need(eg virtlogd is not needed by xen). That's the usage way in alp container and upstream doc too.

Besides, would you please add virtnwfilterd? That's one of the core daemons. Even if we do not have dedicated VT test for it, at least we can cover start/stop the service which is the fundamental but most important check point actually :)

Reply:
These daemons are not meant to be start/stop/restarted at here. The function is to add some settings to the configuration files of all the modular libvirt daemons. So virtlockd and virtlogd daemons theirselves are not touched.

I can add virtnwfilterd as it is required, including daemons and configuration files.

Reply:
> For virtlockd and virtlogd, suggest to let other virt{drv}d start/stop/restart them by dependency need(eg virtlogd is not needed by xen). That's the usage way in alp container and upstream doc too.

Sorry if any confusion. I gave it here because I was considering restarting libvirt daemons as needed , based on https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/16625#discussion_r1189650705. So this is for below line 153, but not for setting debug logs in *.conf.